Uncomment the deb-src lines in /etc/apt/sources.list

paride opened the pull request:
Other minor changes:
 - Run apt-get using eatmydata for performance
 - Do a 'full-upgrade --auto-remove --purge' before attempting the builds
 - Run the builds in an ephemeral container
 - Run `apt-get` in quiet mode

paride committed:
Uncomment the deb-src lines in /etc/apt/sources.list

With no deb-src lines the `apt-get build-dep` stage fails.

paride committed:
Do a 'full-upgrade --auto-remove --purge' before attempting the builds

paride committed:
Run apt-get using eatmydata

eatmydata disables fsync() and other data-to-disk synchronization calls,
dramatically speeding up the installation and configuration of packages.

paride committed:
Run the builds in an ephemeral container

powersj commented on build-packages:
```diff
@@ -72,14 +72,17 @@ launch_container() {
     fi
 
-    lxc launch ubuntu-daily:"$release" "$NAME" ||
+    lxc launch ubuntu-daily:"$release" "$NAME" --ephemeral ||
```
what does this gain us?

paride replied:
Hopefully nothing. But if `lxc delete --force` fails because of ZFS issues (or anything else), the stale STOPPED container will be auto-deleted at the next reboot.

paride committed:
Run apt-get in quiet mode

A single -q produces output suitable for logging, omitting progress indicators.
